src/components/Table.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react'
+import Draggable from 'react-draggable'
+import Row from './Row'
+import EditRow from './EditRow'
+import '../css/Table.css'
+
+class Table extends Component {
+  constructor (props, context) {
+    super(props, context)
+
+    this.displayRows = this.displayRows.bind(this)
+    this.displayTableName = this.displayTableName.bind(this)
+    this.displayTableOptions = this.displayTableOptions.bind(this)
+    this.handleDrag = this.handleDrag.bind(this)
+    this.handleStop = this.handleStop.bind(this)
+    this.handleUpdateName = this.handleUpdateName.bind(this)
+    this.saveName = this.saveName.bind(this)
+    this.toggleEditTable = this.toggleEditTable.bind(this)
+  }
+
+  displayRows (row, index) {
+    return row.edit
```

Comment:
Learned this on recently! Long ternaries can be more easily read like:

```
return row.edit ? (
    <EditRow.......blah blah>
) : (
    <Row....... blah blah>
);
```
it makes it a lot easier to track them.